File: Proyectos Java/Python/Recursividad_ContarDigitos_INF131.py
#Cristian J Velasco Conde
#13759085 LP

def ContarDigitos(n):
    cont = 0
    cont += 1
    if(n==0):
        return 0
    else:
        return(cont+ContarDigitos(n//10))

#===== MAIN ======
print("Ingrese un numero: ")
n=int(input())
print("La Cantidad de Digitos de ",n," es: ",ContarDigitos(n))

File: Proyectos Java/src/Recursividad_ContarDigitos_INF131.java
import java.util.Scanner;

public class Recursividad_ContarDigitos_INF131 {
    public static void main(String[] args){
        //Cristian J. Velasco Conde
        //13759085 LP
        Scanner leer = new Scanner(System.in);
        System.out.println("Ingrese un numero: ");
        int n=leer.nextInt();
        System.out.println("La cantidad de Digitos de "+n+" es: "+ContarDigitos(n));
    }

    public static int ContarDigitos(int n){
        int cont=0;
        cont++;
        if(n==0){
            return 0;
        }else{
            return(cont+ContarDigitos(n/10));
        }
    }
}
